### ISSUE TITLE
Warmup cache

### ISSUE THREAD

**Issue by Qubadi** [optimocha/speed-booster-pack#68]
Hello I have written to you before also about your plugin Speed ​​Booster and it is the best I have used. I have used more than 10 different cache plugins, but none of them compare to Speed ​​Booster. It's amazing how fast the website is. I have installed Speed ​​Booster plugins on all my clients website and they are very happy with it. I'm just wondering about something, and hope you can guide me. When I click on Warmup cache, the web page becomes very very fast, but after a while it gets deleted again. I don't know why it gets slow after a few hours. But when I log into the Wordpress Dashboard and click on Warmup cache, it becomes very fast again, then after a few hours it becomes very slow again. I have enable Warmup cache as well, so I don't know what makes it slow after a few hours. Really hope you can help me and give me a bit more information about this. Thank you

**Comment by optimocha** [comment]
Hey @Qubadi,

Can you tell me what is your setting at cache expiration? It's 10 hours by default, which means cache files will be deleted if they're older than 10 hours.

**Comment by Qubadi** [comment]
Hello again

I have set it to 6 hours, but after 4,5 hours the websites come slow again, so i have manuel go in to Dashboard and click Warmup cache again, and the website come very fast again.

I do not what is wrong, i think its a issue, can u please check it from your side. Thank u again

Qubadi

Sendt fra Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Fra: Optimocha ***@***.***>
Sendt: Saturday, April 22, 2023 10:43:56 PM
Til: optimocha/speed-booster-pack ***@***.***>
Kopi: Qubadi ***@***.***>; Mention ***@***.***>
Emne: Re: [optimocha/speed-booster-pack] Warmup cache (Issue #68)


Hey @Qubadi<https://github.com/Qubadi>,

Can you tell me what is your setting at cache expiration? It's 10 hours by default, which means cache files will be deleted if they're older than 10 hours.

**Comment by optimocha** [comment]
Hi @Qubadi,

I need you to run some test for me, with some easy steps:

1. Warm up the cache.
2. Wait for 5 hours without clearing the cache or warming up the cache again.
3. Using a FTP client or any other file manager of your choice, head to the /wp-content/cache/speed-booster/ directory and check if there are any files there.

If you don't see any files, it means the cache is cleared. There might be another explanation for this, like some other plugin or another action (like publishing a post) clears the cache over and over again before the current cache goes stale.

Final question: Is the "Warm up cache on clear" setting also enabled?

**Comment by Qubadi** [comment]
Hello

Thank you very much for the quick feedback.
I have a cache time on Speed Booster of 6 hours, I have also clicked on delete cache and Warmup cache additions.

I am sending you some pictures, I hope we can find a solution to this. Many thanks in advance, and greatly appreciate the help.

Greeting
Qubadi

________________________________
Fra: Optimocha ***@***.***>
Sendt: søndag 23. april 2023 21:18
Til: optimocha/speed-booster-pack ***@***.***>
Kopi: Qubadi ***@***.***>; Mention ***@***.***>
Emne: Re: [optimocha/speed-booster-pack] Warmup cache (Issue #68)


Hi @Qubadi<https://github.com/Qubadi>,

I need you to run some test for me, with some easy steps:

  1.  Warm up the cache.
  2.  Wait for 5 hours without clearing the cache or warming up the cache again.
  3.  Using a FTP client or any other file manager of your choice, head to the /wp-content/cache/speed-booster/ directory and check if there are any files there.

If you don't see any files, it means the cache is cleared. There might be another explanation for this, like some other plugin or another action (like publishing a post) clears the cache over and over again before the current cache goes stale.

Final question: Is the "Warm up cache on clear" setting also enabled?

**Comment by Qubadi** [comment]
Images:

https://i.postimg.cc/fRzNQpb9/Speed-Booster-1.png
https://i.postimg.cc/W4tchZgp/Speed-Booster-2.png
https://i.postimg.cc/9QLjmb4N/Speed-Booster-3.png
https://i.postimg.cc/SK5F8Kf6/Speed-Booster-4.png

**Comment by optimocha** [comment]
Hey @Qubadi, sorry for the late response. I've seen your images and everything seems to be working properly. If you're still having slowness issues even when the /cache/ folder is populated, it means that the server response time is high even while the pages are cached. If that's the case, you should contact your server administrator to troubleshoot server response issues.

**Comment by Qubadi** [comment]
Hello

Thank's for your feedback. I greatly appreciate your time and help.

As I said, I have used the same cache plugins for other websites, and they work perfectly fine, only on the one website that does not work as it should.

The problem is, when I go to File manager and check the cache folder, it only says index.html, and when I click on Warmup cache manually, all the pages in the cache folder appear. As I said, this only happens on one of those websites, the others work perfectly fine.

Have contacted the Hosting provider, they could not see anything wrong on their side.

Then there is probably a plugin on the website that makes the cache plugins not work as they should.

Thank you again, I wish you a lovely day.

**Comment by Qubadi** [comment]
NEW UPDATE

We found out what was the problem. We were informed by our Hosting provider that there are a number of plugins that prevent the website from being cached. Some of the plugins had set-cookie active, and that meant that the cache plugins did not work as it should, and there was nothing in the folder cache, it became empty after a few hours, and as I said the reason was set-cookie. I have now fixed this and the plugin works great. Thank you so much.
